data/spells/scripts/attack/lightest_magic_missile_rune.lua

```diff
@@ -4,7 +4,7 @@ combat:setParameter(COMBAT_PARAM_EFFECT, CONST_ME_ENERGYAREA)
 combat:setParameter(COMBAT_PARAM_DISTANCEEFFECT, CONST_ANI_ENERGY)
 
 function onGetFormulaValues(player, level, maglevel)
-	return -3, -5
+	return -3, -7
```

Comment:
Unfortunately, we can't really much on *wikia* on these kind of stuff. I learned much following/reading Gorak first researches of spells from vanilla, so I'm much involved on this aspect of this game xD.

The current constants (formulas) for these spells were changed when I did the same research again on Feb. 2017. I was waiting for the right moment to use them here.

By the way, we can't even the spell Practice Magic Missile Rune on vanilla anymore, weird xD.